.cloudbuild/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM   gcr.io/cloud-builders/bazel
```

Comment:
Same here, you cannot just use the last version. At least not until bazel is GA.

Reply:
Can we just use envoyproxy/envoy-build as base image, maybe with SHA tagged. That ensure we meet envoy build requirements when we update it.

Reply:
Good point - however Shriram recommended to use the same image with istio/istio. But we may base that one on envoyproxy. 

Only problem is that envoy-build has even more deps (clang, etc).

For now - I'm removing cloudbuild, it didn't work, will let others handle it...